chipsec/testcase.py

```diff
@@ -347,7 +337,7 @@ def print_summaryL(self, runtime: Optional[float] = None) -> None:
                         logger().log_information(mod)
                     elif result == 'failed':
                         logger().log_failed(mod)
-                    elif result in ['not applicable', 'skipped']:
+                    elif result in 'not applicable':
```

Comment:
Should either be 
```suggestion
                    elif result in ['not applicable']:
```
Or 
```suggestion
                    elif result == 'not applicable':
```